.github/workflows/certification.yml

```diff
@@ -245,10 +252,65 @@ jobs:
           exit 1
         fi
 
+    - name: Prepare Cert Coverage Info
+      run: |
+       mkdir -p path/to/cov_files
```

Comment:
```suggestion
       mkdir -p /tmp/coverage_data
```

Reply:
`Path to` seems fine to me, so not changing - unless I am violating any standard.

Reply:
Fine to change to /temp as well, as linux wise, that may sound conventional. Will do in next commit.